ocs_ci/ocs/ocp.py

```diff
@@ -428,7 +447,10 @@ def wait_for_resource(
 
                 # Only 1 resource expected to be returned
                 if resource_name:
-                    status = self.get_resource(resource_name, column)
+                    retry = int(timeout / sleep if sleep else timeout / 1)
```

Comment:
I'm not sure what is executed first but please check that timeout / sleep if sleep, will not fail if sleep is 0 as it will be dividing by 0

Reply:
This line is here exactly for preventing this issue ;)